Cargo.toml

```diff
@@ -1,14 +1,17 @@
 [package]
-name = "imap-proto"
```

Comment:
I prefer to keep the common order that most packages seem to have, rather than this alphabetized order.

Reply:
I prefer to follow a well-defined ordering everywhere possible, but changed it back.